contracts/avalanche/AvalancheTokenController.sol

```diff
@@ -449,11 +449,11 @@ contract AvalancheTokenController {
         registryAdmin = admin;
     }
 
-    function setIsMintPauser(address account, bool status) external onlyRegistryAdmin {
+    function setIsMintPauser(address account, bool status) external onlyOwner {
         isMintPauser[account] = status;
     }
 
-    function setIsMintRatifier(address account, bool status) external onlyRegistryAdmin {
+    function setIsMintRatifier(address account, bool status) external onlyOwner {
```

Comment:
Are these changes we'd like to backport to mainnet TUSD? If so, let's create a task and/or submit a separate PR.

Reply:
Yes. I looked over some of the contracts and there are some issues in mainnet TUSD as well.